libfuzzer/run_corpus.py

```
@@ -0,0 +1,155 @@
+# Copyright (c) uBPF contributors
+# SPDX-License-Identifier: MIT
+
+# This script runs two bpf_conformance plugins against a corpus of bpf programs
+# and compares the results. It is used to verify that the plugins are equivalent.
+# Each plugin consists of a binary and a set of plugin specific options.
+
+import os
+import argparse
+import subprocess
+from typing import List, Optional
```

Ruff

`typing.List` imported but unused

Remove unused import

(F401)